docs/misc/job-order.md

```diff
@@ -1,18 +1,18 @@
-你有 $n$ 个任务，要求你找到一个代价最小的的顺序执行他们。第 $i$ 个任务花费的时间是 $t_i$ ，而第 $i$ 个任务等待 $t$ 的时间会花费 $f_i(t)$ 的代价。
+Assume you have $n$ tasks, and you are required to find a least costly order to execute them. The time spent by the $i$-th task is $t_i$ , and the time spent by the $i$-th task waiting for $t$ is $f_i(t)$ .
```

Comment:
```suggestion
Assume you have $n$ tasks, and you are required to find a order with least cost to execute them. The time spent on the $i$-th task is $t_i$ , and the time spent by the $i$-th task waiting for $t$ is $f_i(t)$ .
```
It costs $t_i$ to finish $i$-th job and $f_i(t)$ to make this job wait for time $t$.